app/controllers/home_controller.rb

```diff
@@ -18,15 +18,15 @@ def reference_calculator
       @bank_accounts = @types.includes(:bank_account).map(&:bank_account).uniq.compact
       @bank_accounts = [BankAccount.last] if @bank_accounts.empty?
     else
-      redirect_to root_url, alert: I18n.t('group_orders.errors.no_member')
+      redirect_to root_path, alert: I18n.t('group_orders.errors.no_member')
```

Comment:
The PR says this is about adding a test, not about changing redirect behavior. Has this maybe slipped in from some other branch? (Other than that I'm fine with it!)

Reply:
I tried to make the usage of  _path and _url uniform, but no need to do it here...
The change in l67 is actually a fix see https://github.com/foodcoops/foodsoft/pull/970#discussion_r1039455645